Validation/DTRecHits/python/DTRecHitClients_cfi.py

```diff
@@ -1,8 +1,22 @@
 import FWCore.ParameterSet.Config as cms
 
-dtrechitclients = cms.EDAnalyzer("DTRecHitClients")
-dt2dsegmentclients = cms.EDAnalyzer("DT2DSegmentClients")
-dt4dsegmentclients = cms.EDAnalyzer("DT4DSegmentClients")
+dtrechitclients = cms.EDProducer("DTRecHitClients",
```

Comment:
Note: these should go via https://github.com/cms-sw/cmssw/blob/master/DQMServices/Core/python/DQMEDHarvester.py . Not a functional problem for now, and likely not for the future, but a matter of keeping some order in the plugin types.